fiftyone/operators/executor.py

```diff
@@ -501,6 +504,30 @@ def view(self):
 
         return self._view
 
+    def target_view(self, param_name="view_target"):
```

Comment:
Should we call it `param_name="VIEW"` for consistency?

Reply:
Thats the value - this is choosing which param name - `ctx.params[param_name]`

Reply:
Ohh, yah true. Never mind 🙈